src/helpers/elements-interactions.ts

```diff
@@ -36,10 +36,10 @@ async function fillInput(pageOrFrame: Page | Frame, inputSelector: string, input
 }
 
 async function setValue(pageOrFrame: Page | Frame, inputSelector: string, inputValue: string): Promise<void> {
-  await pageOrFrame.$eval(inputSelector, (input: Element, inputValue) => {
+  await pageOrFrame.$eval(inputSelector, (input: Element, value) => {
     const inputElement = input;
     // @ts-ignore
-    inputElement.value = inputValue;
+    inputElement.value = value;
   }, [inputValue]);
```

Comment:
Why did you renamed?

(I don't really care, but it is very hard to review this PR with a lot of meaningless changes)

Reply:
it is not meaningless, there was an eslint error for re defining the same variable name from outer scope. 
You can see there a re 2 variables named "value", one in the outer function signature and one in the inner function signature. In this specific case its harmless but can lead to errors or issues in the future :)

Reply:
Oh OK sorry, I knew this is the reason but I missed the outer `inputValue`.